main.c

```diff
@@ -98,17 +100,30 @@ void setup_signal(int signum, void (*handler)(int)) {
 	sigact.sa_flags = 0;
 	sigact.sa_handler = handler;
 	sigemptyset(&sigact.sa_mask);
-	
+
 	if (sigaction(signum, &sigact, NULL) < 0)
 		error(0, errno, "signal %d", signum);
 }
 
+void run_command(const char* cmd) {
+    cmdpid = fork();
+    if (cmdpid < 0) {
+        error(EXIT_FAILURE, errno, "fork");
+    } else if (cmdpid > 0) {
+        wait(NULL);
+    } else {
+        execlp("sh", "sh", "-c", cmd, NULL);
```

Comment:
When running `physlock` in a systemd service, these `-a` and `-b` flags won't work. I'm getting the following error:
```
physlock: exec: No such file or directory
```

This can be tested as:

```
$ PATH="" /path/to/bin/physlock -b "/path/to/bin/echo HELLO"
physlock: exec: No such file or directory
```

I guess it cannot find `sh` because the environment is very clean in systemd services.

Instead of using `"sh"` and letting `execlp` to find it, could it just use hardcoded path `"/bin/sh"`? I guess that should always be the path.

Reply:
I don't think any distro has dropped `/bin/sh` (at the very minimum it exists as a symlink), so this seems reasonable.